packages/hoppscotch-sh-admin/src/composables/useConfigHandler.ts

```diff
@@ -86,10 +93,17 @@ export function useConfigHandler(updatedConfigs?: Config) {
       configNames: [
         'GOOGLE_CLIENT_ID',
         'GOOGLE_CLIENT_SECRET',
+        'GOOGLE_CALLBACK_URL',
+        'GOOGLE_SCOPE',
         'MICROSOFT_CLIENT_ID',
         'MICROSOFT_CLIENT_SECRET',
+        'MICROSOFT_CALLBACK_URL',
+        'MICROSOFT_SCOPE',
+        'MICROSOFT_TENANT',
         'GITHUB_CLIENT_ID',
         'GITHUB_CLIENT_SECRET',
+        'GITHUB_CALLBACK_URL',
+        'GITHUB_SCOPE',
         'MAILER_SMTP_URL',
         'MAILER_ADDRESS_FROM',
         'ALLOW_ANALYTICS_COLLECTION',
```

Comment:
We could combine these entries separately specific to each provider so that they can be consumed at other places and merged here. We can revisit this separately.